packages/terra-menu/src/Menu.module.scss

```diff
@@ -46,6 +46,10 @@
       word-wrap: normal;
     }
 
+    .main-header-title {
+      margin-left: var(--terra-menu-header-title-margin-left, 10px);
```

Comment:
What happens if the heading is long? Should this have padding on both sides?

Reply:
Actually, why does the main header have unique styling? Should it not be the same as the sub menus?

Reply:
> What happens if the header title is long? Should this have padding on both sides?

That's a good question. Currently ,if the sub-menu header title is longer than the width of the menu, an ellipsis is shown at the end of the title(a non-functioning ellipsis ). And the same logic applies to main menu header-title as well at this point. 

@jeremyfuksa what are your thoughts for longer header-titles ? Should we display the complete text in the header-title section or should we enable a functioning ellipsis ?

Reply:
@lokesh-0813 Looking at the issue you linked, bullet number five says:
>  either refactor top header(and submenu nested menu headers) so the height will flex based on title content and correctly wrap, or import action-header which already has intended layout.